homeassistant/components/lutron/config_flow.py

```diff
@@ -0,0 +1,88 @@
+"""Config flow for Lutron integration."""
+
+import logging
+from typing import Any, Dict, Optional
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import CONN_CLASS_LOCAL_PUSH, ConfigFlow, OptionsFlow
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import callback
+from homeassistant.helpers.typing import ConfigType
+
+from .const import (  # pylint:disable=unused-import
+    CONF_ENABLE_AREAS,
+    DEFAULT_ENABLE_AREAS,
+    DOMAIN,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class LutronConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Config flow for Lutron RadioRA 2."""
+
+    VERSION = 1
+    CONNECTION_CLASS = CONN_CLASS_LOCAL_PUSH
+
+    async def async_step_import(
+        self, user_input: Optional[ConfigType] = None
+    ) -> Dict[str, Any]:
+        """Handle a flow initiated by configuration file."""
+        return await self.async_step_user(user_input)
+
+    async def async_step_user(
+        self, user_input: Optional[ConfigType] = None
+    ) -> Dict[str, Any]:
+        """Handle a flow initiated by the user."""
+        if user_input is None:
+            return self._show_setup_form()
+
+        await self.async_set_unique_id(user_input[CONF_HOST])
```

Comment:
Ip address is not allowed as unique id. Is this an ip address?

Reply:
If there's no other good unique id, we can just check the existing config entries and make sure we don't set up another entry with the same host. Ie do this without using the unique id api.